### PR TITLE
grid component

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Grid:
- grid of 2 items
- grid of 3 items
- grid of 4 items

### CODE REVIEW

File: grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+.div4 {
+    border: 2px solid var(--WHITE);
+    background-color: var(--GRAY-60);
+}
+
+.grid-three {
+    display: grid;
+    grid-template-columns: 1fr 60rem;
```
Comment (nikhiljugale007) [inline grid/grid.css]: For grid three I guess we have to make 3 columns, here you are given only 2 columns. I am not able to understand.
I think you can use grid-template-column: 1fr 1fr 1fr.

**Review by nikhiljugale007 — COMMENTED** [review]
UI is good.
Added suggestion in code.

File: grid/grid.css (hunk 1 above)
Comment (swstica) [reply grid/grid.css]: +1

**Comment by swstica** [comment]
Rest all looks pretty good. Can be merged!!

File: grid/grid.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply grid/grid.css]: Thanks I'll resolve it.

File: grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+.div1,
+.div2,
+.div3,
+.div4 {
+    /* max-width: 30%;
+    width: auto; */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 20px 0;
```
Comment (theVivek-kumar) [inline grid/grid.css:30]: margins can be in rem

File: grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+.div2 {
+    border: 2px solid var(--WHITE);
+    background-color: var(--BLACK-60);
+}
+
+.div3 {
+    padding: 4rem 2rem;
+    border: 2px solid var(--WHITE);
```
Comment (theVivek-kumar) [inline grid/grid.css:51]: use rem

File: grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    background-color: var(--MAROON-80);
+    grid-area: div-3;
+}
+
+.div-4{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 2px solid var(--WHITE);
+    background-color: var(--GRAY-60);
```
Comment (theVivek-kumar) [inline grid/grid.css:115]: use rem instead of px

File: grid/grid.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                    </li>
+                </ul>
+            </div>
+
+            <div class="divider"></div>
+
+            <div class="aside--header">
+                <h3 class="aside--title">Components</h3>
+            </div>
+            <div class="aside--body">
```
Comment (theVivek-kumar) [inline grid/grid.html:72]: spacing is good

**Review by theVivek-kumar — APPROVED** [review]
can be merged

**Pull request merged**